cmd/run/run.go

```diff
@@ -293,6 +293,18 @@ func DefaultConfig() *Config {
 	}
 }
 
+// TCPRandomPort returns a random TCP Port along with a function that releases the port.
+// It is the responsibility of the caller to release the port. Otherwise, the port will be orphaned.
```

Comment:
```suggestion
// TCPRandomPort tries to find a random TCP Port. If it can't find one, it panics. Else, it returns the port and a function that releases the port.
// It is the responsibility of the caller to call the release function.
```